docs/notebooks/binning_slowly_changing_sources.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ens.bin_sources(time_window=7.0, offset=0.0)
</pre><p>What happens if you bin with non-integer value? We basically do not want that, right? Perhaps this should be forced to be an int?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/lsstseries/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
This should still work with a non-integer value for `time_window`. The concept of an `offset` is a bit weird at less than a 24 hour binning. In this case `offset` still shifts the bins but it does not need to be aligned with the middle of the daylight hours.

Reply:
Hm... Should we maybe make a warning if you are trying do time_window with floats? I am worried about a case where algorithm silently make results that users may not expect.

Reply:
I don't think that we need a warning. The algorithm will still bin correctly. I imagine 1 hour binning will likely be a valid (and useful) case.